### DISCUSSION TITLE
Приветствие

### DISCUSSION THREAD

**Discussion by StarDust0704** [community/community#148178] [New to GitHub]
Привет я совсем зеленый но есть огромное желание изучить профессию Python-разработчика
Если буду где то ошибаться прошу сильно не пинать

**Comment by ghostinhershell** [comment]
Hi @StarDust0704, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
